python/ray/serve/tests/test_batching.py

```diff
@@ -291,6 +292,84 @@ async def func(key1, key2):
     assert result == [("hi1", "hi2"), ("hi3", "hi4")]
 
 
+@pytest.mark.asyncio
```

Comment:
can you also add an integration test that uses this w/ `StreamingResponse`s?

It should still work without my PR merged, will just not actually stream the results back to end client iteratively.

Reply:
I added `test_batch_generator_streaming_response_integration_test` in `test_batching.py`.